### ISSUE TITLE
Due Date

### ISSUE THREAD

**Issue by jgmac1106** [jgmac1106/bloggingme#4]
Going to shoot for 11/2.

**Comment by jackjamieson2** [comment]
Sounds good — This week is a little busy but I'll make sure I can devote some time to this paper.

I can definitely contribute to the section on IndieWeb's history, and probably to its shared community values.  I've got some time for writing tomorrow, so I'll try to write some of those sections then. 

If you've posted anything for those sections in the DS106 study I'll use those as a reference for structure. Otherwise, I'll write what I can and assume it will need editing.

(And obviously that goes for everywhere — feel free to edit/remove anything you think should change!)

**Comment by jgmac1106** [comment]
Writing all day when I get to office. There will be a template for you but
you have also contributed plenty if time crunch.

**Comment by jackjamieson2** [comment]
Looking great! I'm going to create a new branch so I can edit without fear of overwriting any of your text.  I (finally) have a bit of time to write today, so I'm going to add a few notes to the indieweb case study

**Comment by jgmac1106** [comment]
awesome, I will still away from that file for awhile and move onto the
discussion and conclusion.

**Comment by jackjamieson2** [comment]
just pushed the jj_edits branch with additions to indiewebcasestudy.md and references.md

https://github.com/jgmac1106/bloggingme/tree/jj_edits

**Comment by jackjamieson2** [comment]
Ah, just saw your reply. I'll merge my branch to keep things together

**Comment by jackjamieson2** [comment]
Merged!  Might be all I can do today, so I hope its helpful!

**Comment by jackjamieson2** [comment]
Since I saw you're still editing the indieweb case study I just added a couple rough thoughts.  I can elaborate if you want

**Comment by jgmac1106** [comment]
yeah done right better than done on time..gonna try and finish today

**Comment by jgmac1106** [comment]
This is probably so over word count already, but I need the chapters for
Open Pedagogy book I need to finish so anything will ditch will be chapters
there.
